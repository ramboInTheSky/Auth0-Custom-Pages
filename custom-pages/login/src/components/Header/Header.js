import React from 'react'
import PropTypes from 'prop-types'
import './Header.scss'

export const Header = props => (
  <div className={props.className}>
    <h1>NICE accounts</h1>
    <h6>A single sign-in for the different services offered by NICE.</h6>
  </div>
)

Header.propTypes = {
  className: PropTypes.string
}

Header.defaultProps = {
  className: ''
}

export default Header
