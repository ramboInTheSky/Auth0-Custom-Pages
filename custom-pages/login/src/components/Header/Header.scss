@import '../variables.scss';

h1{
  font-weight: 900;
    font-style: normal;
    font-size: 50px;
}

h6{
  font-weight: 700;
}