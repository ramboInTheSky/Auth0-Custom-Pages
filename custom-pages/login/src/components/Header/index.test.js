
import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-14'

import Header from './'

Enzyme.configure({ adapter: new Adapter() })

describe('Header components', () => {
  it('should render <Header /> correctly', () => {
    expect(shallow(<Header />)).toMatchSnapshot()
  })
})
