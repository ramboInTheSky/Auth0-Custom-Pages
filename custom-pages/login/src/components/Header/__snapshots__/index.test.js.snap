// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header components should render <Header /> correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    className=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          NICE accounts
        </h1>,
        <h6>
          A single sign-in for the different services offered by NICE.
        </h6>,
      ],
      "className": "",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "NICE accounts",
        },
        "ref": null,
        "rendered": "NICE accounts",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "A single sign-in for the different services offered by NICE.",
        },
        "ref": null,
        "rendered": "A single sign-in for the different services offered by NICE.",
        "type": "h6",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            NICE accounts
          </h1>,
          <h6>
            A single sign-in for the different services offered by NICE.
          </h6>,
        ],
        "className": "",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "NICE accounts",
          },
          "ref": null,
          "rendered": "NICE accounts",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "A single sign-in for the different services offered by NICE.",
          },
          "ref": null,
          "rendered": "A single sign-in for the different services offered by NICE.",
          "type": "h6",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFourteenAdapter {
      "options": Object {
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "prevContext": true,
          },
          "getChildContext": Object {
            "calledByRenderer": true,
          },
        },
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
